lib/setupmanagers/qemuhck/qemuhck.rb

```diff
@@ -108,6 +111,33 @@ def initialize_clients_vm
       end
     end
 
+    def write_host_info
+      collect_vms_info
```

Comment:
It's odd to collect "VM" information when writing host information.
Why will you want to log VM information anyway? AutoHCK already logs the whole QEMU command line.

Reply:
`AutoHCK.log` are uploaded only at the end, we want to have this info during the CI run to easily know how to connect to the running env.